libs/components/config/src/lib/params.ts

```diff
@@ -150,25 +155,44 @@ export class SkyAppRuntimeConfigParams {
   }
 
   /**
-   * Adds the current params to the supplied url.
+   * Adds the current params to the supplied URL.
    */
   public getUrl(url: string): string {
+    return this.#excludeParamsFromUrl(url, this.#excludeFromRequestsParams);
+  }
+
+  /**
+   * Adds the current params to the supplied link URL.
+   */
+  public getLinkUrl(url: string): string {
+    return this.#excludeParamsFromUrl(
+      url,
+      this.#excludeFromRequestsParams.concat(this.#excludeFromLinksParams)
+    );
+  }
+
+  #excludeParamsFromUrl(url: string, excludeParams: string[]): string {
```

Comment:
The name `#excludeParamsFromUrl()` is a little confusing because the function also includes other params. Maybe `#buildUrlWithParams()` and let the `excludeParams` parameter name indicate that some will be excluded?

Reply:
Good callout. I refactored the loop and renamed the method.